frontend/src/app/admin/superadmin/structures/structures.component.scss

```
@@ -65,52 +65,124 @@
+    transform: translateY(-2px);
+    box-shadow: 0 20px 34px rgba(15, 23, 42, 0.12);
+    border-color: #bfdbfe;
   }
+}
 
-  .structure-header {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin-bottom: 10px;
+.structure-tile__head {
```

Stylelint (17.2.0)

[error] Expected class selector ".structure-tile__head" to be kebab-case (selector-class-pattern)

(selector-class-pattern)